Marlin/pins_AZTEEG_X3.h

```diff
@@ -75,10 +74,8 @@
 //
 // Misc
 //
-#if ENABLED(OK_TO_CHANGE_CASE_LIGHT) && STAT_LED_RED_PIN == 6
+#if ENABLED(CASE_LIGHT_ENABLE)  && PIN_EXISTS(CASE_LIGHT) && PIN_EXISTS(STAT_LED_RED) && STAT_LED_RED_PIN == CASE_LIGHT_PIN 
```

Comment:
Can you imagine if every feature added this kind of conditional to every pins file? I'm sure we can handle this kind of thing outside of the pins files, somehow…. We've used the concept of `DEFAULT_THING_PIN` which provides just the default pin that would be used for some thing, then later the logic around the feature decides to override it, or…

Just as we currently have the concept of every board having a more-or-less standard order of using the MOSFETs, so that we can simply define the MOSFETs and the function depends on what is enabled… I think we could define a set of GPIO and Analog pins on every board which is left over for other functions, then define which of those leftover pins should be used for each function. It might not yield a universal standard for pin usage, but it would give us a sense of what pins are left over on each board, and what the limits are for adding features to each one.

Reply:
Is anyone able to help me implement this fix?  I'm a newb and am having trouble with it.  I downloaded the h files and overwrote what was on my system, but it's causing even more issues.  With the newest bugfix build, I'm getting a number of these errors:

```
Marlin\src\HAL\AVR\../../inc/../pins/ramps/pins_AZTEEG_X3.h:75:70: error: macro "PIN_EXISTS" passed 2 arguments, but takes just 1
 #if ENABLED(CASE_LIGHT_ENABLE) && PIN_EXISTS(CASE_LIGHT, STAT_LED_RED) && STAT_LED_RED_PIN == CASE_LIGHT_PIN
```
Any and all help would be greatly appreciated.